src/main.c

```diff
@@ -55,16 +93,31 @@ int main()
 	draw2fb(fb[1], 1, 8*10);
 	draw2fb(fb[1], 2, 8*11);
 	draw2fb(fb[1], 3, 8*12);
+	fb_num = 2;
 
-	TMR0_TimerInit(1500);
+	TMR0_TimerInit(SCAN_T);
 	TMR0_ITCfg(ENABLE, TMR0_3_IT_CYC_END);
 	PFIC_EnableIRQ(TMR0_IRQn);
 
-	while (1) {
-	}
+	btn_init();
+	btn_onOnePress(KEY1, change_mode);
+	btn_onOnePress(KEY2, change_fb);
+	btn_onLongPress(KEY1, change_brightness);
+
+    while (1) {
+		int i = 0;

```

Comment:
unsigned

Reply:
you will need to hold the button for 13.6102208 years to get this int overflowed =]